internal/util/importutil/binlog_adapter.go

```diff
@@ -43,13 +44,14 @@ type SegmentFilesHolder struct {
 // 1. read insert log of each field, then constructs map[storage.FieldID]storage.FieldData in memory.
 // 2. read delta log to remove deleted entities(TimeStampField is used to apply or skip the operation).
```

Comment:
Here and other places:
Please note that doc comments for packages, functions and types should start with complete sentences naming the declared symbol.

For example here:
`Comment should have the following format 'BinlogAdapter ...' (with an optional leading article) `